src/Sentry/GlobalSessionManager.cs

```diff
@@ -145,7 +151,8 @@ public GlobalSessionManager(SentryOptions options)
 
                 var id =
                     TryGetPersistentInstallationId() ??
-                    TryGetHardwareInstallationId();
+                    TryGetHardwareInstallationId() ??
+                    Environment.MachineName;
```

Comment:
Could you please you `SHA256` this or something? So we don't rely on the raw machine name?

Reply:
Good point, missed that

Reply:
Done